From 24690e5e98934faf846531f2515b03cc53f5d0c9 Mon Sep 17 00:00:00 2001
From: Yumi Yukimura <me.cafebabe@gmail.com>
Date: Mon, 24 Jun 2024 23:18:55 +0800
Subject: [PATCH] compatibility_matrices: Allow radio 1.5 on target-level 6

* I do expect 1.5 support will get deprecated in framework, the
  purpose of this is for Radio HIDL->AIDL wrapper to use

Change-Id: I5baf60d94affef649d68ef3f076539e04221ed41
---
 compatibility_matrices/compatibility_matrix.6.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/compatibility_matrices/compatibility_matrix.6.xml b/compatibility_matrices/compatibility_matrix.6.xml
index fdd7952503..4c375ce690 100644
--- a/compatibility_matrices/compatibility_matrix.6.xml
+++ b/compatibility_matrices/compatibility_matrix.6.xml
@@ -450,7 +450,7 @@
     </hal>
     <hal format="hidl">
         <name>android.hardware.radio</name>
-        <version>1.6</version>
+        <version>1.5-6</version>
         <interface>
             <name>IRadio</name>
             <instance>slot1</instance>
